tests/ProfilerTest.php

```diff
@@ -30,7 +30,7 @@ public function test__construct()
         $profiler = self::makeProfiler();
         $uuid = $profiler->push();
         $this->assertIsString($uuid);
-        $this->assertSame($profiler->uuid->getValue(), $uuid);
+        $this->assertSame($profiler->getId(), $uuid);
```

Comment:
The `Profile::$uuid` is `private` variable now. Using the `Profile::getId()` method to get private `uuid` varirable value.